### PR TITLE
Feature/jm-7329: As a court officer I want to generate a manual juror report (FE)

### PR DESCRIPTION
**Pull request by hari-vajja**
### JIRA link (if applicable) ###
https://centralgovernmentcgi.atlassian.net/browse/JM-7329

### Change description ###
Added functionality to generate a manual juror report

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: server/routes/reporting/standard-report/definitions.js [hunk 1]
```diff
@@ -405,6 +405,19 @@
...
+        search: 'dateRange',
+        headings: [
+          'dateFrom',
+          'reportDate',
+          'dateTo',
+          'reportTime',
+          'totalManuallyCreatedJurors',
+          'courtName',
+        ],
+      },
```
Comment (ricardofreitasrocha) [inline server/routes/reporting/standard-report/definitions.js:420]: this report needs to be printed in landscape mode with: `landscape: true`

File: client/templates/reporting/court-reports.njk [hunk 2]
```diff
@@ -45,7 +45,7 @@
       <h2 class="govuk-heading-m">Senior jury officer reports</h2>
       <ul class="govuk-list">
         <li><a class="govuk-link" href="#">Jury ammendment report</a></li>
-        <li><a class="govuk-link" href="#">Manually-created jurors report</a></li>
+        <li><a class="govuk-link" href="{{ url('reports.manual-juror-report.filter.get') }}">Manually-created jurors report</a></li>
       </ul>
```
Comment (ricardofreitasrocha) [inline client/templates/reporting/court-reports.njk:49]: we should hide these links behind `{% isSJO %}`

**Review by ricardofreitasrocha — CHANGES_REQUESTED** [review]

File: client/templates/reporting/court-reports.njk (hunk 2 above)
Comment (hari-vajja) [reply client/templates/reporting/court-reports.njk:50]: Done

File: server/routes/reporting/standard-report/definitions.js (hunk 1 above)
Comment (hari-vajja) [reply server/routes/reporting/standard-report/definitions.js]: Updated

**Review by ricardofreitasrocha — APPROVED** [review]

**Pull request merged**